Nodejs/Setup/swix/Microsoft.VisualStudio.NodejsTools.Targets.swixproj

```diff
@@ -31,7 +31,9 @@
   </Target>
```

Comment:
no need to include it for this change, but think we take care of the vsix copy in BuildRelease, so this is no longer necessary